extensions/Particle3D/PU/CCPUEmitter.h

```diff
@@ -365,206 +919,107 @@ class CC_DLL PUEmitter : public Particle3DEmitter
-        is also possible to set the dimensions per particle. Note, that this only applies to visual particles,
-        of course. This attribute is used to x, y and z dimensions with the value.
-    */
-    PUDynamicAttribute* _dynParticleAllDimensions;
-    bool _dynParticleAllDimensionsSet;
+    PUDynamicAttribute* _dynRepeatDelay;///<Dynamic attribute used to define the repeat/delay of the emitter.
 
-    /** Dynamic attribute used to define the (own) width of a particle.
-    @remarks
-        In the Particle Technique it is possible to set the default width for all particles, but it
-        is also possible to set the width per particle. Note, that this only applies to visual particles,
-        of course.
-    */
-    PUDynamicAttribute* _dynParticleWidth;
-    bool _dynParticleWidthSet;
+    PUDynamicAttribute* _dynParticleAllDimensions;///<Dynamic attribute used to define the (own) dimensions of a particle.
+    bool _dynParticleAllDimensionsSet;///<Whether use dynamic attribute to define the dimensions of a particle.
 
-    /** Dynamic attribute used to define the (own) height of a particle.
-    @remarks
-        In the Particle Technique it is possible to set the default height for all particles, but it
-        is also possible to set the height per particle. Note, that this only applies to visual particles,
-        of course.
-    */
-    PUDynamicAttribute* _dynParticleHeight;
-    bool _dynParticleHeightSet;
+    PUDynamicAttribute* _dynParticleWidth;///<Dynamic attribute used to define the (own) width of a particle.
+    bool _dynParticleWidthSet;///<Whether use dynamic attribute to define the width of a particle.
 
-    /** Dynamic attribute used to define the (own) depth of a particle.
-    @remarks
-        In the Particle Technique it is possible to set the default depth for all particles, but it
-        is also possible to set the depth per particle. Note, that this only applies to visual particles,
-        of course.
-    */
-    PUDynamicAttribute* _dynParticleDepth;
-    bool _dynParticleDepthSet;
+    PUDynamicAttribute* _dynParticleHeight;///<Dynamic attribute used to define the (own) height of a particle.
+    bool _dynParticleHeightSet;///<Whether use dynamic attribute to define the height of a particle.
 
-    /** Notional up vector, just used to speed up generation of variant directions.
-    */
-    Vec3 _upVector;
+    PUDynamicAttribute* _dynParticleDepth;///<Dynamic attribute used to define the (own) depth of a particle.
+    bool _dynParticleDepthSet;///<Whether use dynamic attribute to define the depth of a particle.
 
-    /** Particles that are left to be emitted from the previous time.
-    */
-    float _remainder;
+    Vec3 _upVector;///<Notional up vector, just used to speed up generation of variant directions.
+
+    float _remainder;///<Particles that are left to be emitted from the previous time.
```

Comment:
As you know that the emitted particles must be int. This is the time left by last emit circle which is not enough to emit one particle.